i18n/de/docusaurus-plugin-content-docs/current/account-affiliate.md

```diff
@@ -5,113 +5,100 @@ description: Anleitung wie du Geld mit ZAP-Hosting verdienen kannst, indem du Fr
 
-:::info
-Neben dem Banner wird ein Link erstellt, auf den das Banner verweist. Du kannst deinen Banner-Link anpassen, wie du willst, damit der Link lesbar wird und einfach gemerkt werden kann. Das Alias wird an "https://zap-hosting.com/" angehängt. Der Link sieht dann so aus: 
+Wir empfehlen, die Option **Produkte kaufen** zu aktivieren, da sie den Kunden direkt zum Serverkonfigurator für das ausgewählte Produkt führt und nicht zur Produktseite. Je weniger Seiten ein Nutzer durchlaufen muss, desto wahrscheinlicher ist es, dass er eine Bestellung aufgibt.
 
-"https://zap-hosting.com/alias". 
+Als Nächstes kannst du über das Dropdown-Menü das Feld **Produkt** auswählen. Dorthin wird dein neuer Partnerlink den Nutzer weiterleiten, wenn er ihn aufruft. Wenn du einen Gameserver auswählst, gibt es eine zusätzliche Option **Spiel**, die du auswählen musst, ansonsten ist das Produkt die einzige Option.
 
-Wenn du kein Alias erstellst, wird ein zufälliges Alias erstellt.
-:::
+#### Abschluss des Erstellungsprozesses
+Wenn du alles nach deinen Vorstellungen ausgefüllt hast, drückst du einfach auf die grüne Schaltfläche **Erstellen**, um mit der Erstellung deines Banners oder Links fortzufahren. Du solltest eine Erfolgsmeldung erhalten, wenn alles korrekt ist, und dein neuer Banner oder Link erscheint unter dem Abschnitt **Deine Banner/Links**, wenn du nach unten scrollst.
 
+![image](https://screensaver01.zap-hosting.com/index.php/s/724JJWXKptMB9qR/preview)
 
-`Vorlage`
+## Auszahlen & Übertragen
 
-:::info
-Hier wählst du die Vorlage aus einer Reihe verschiedener Banner aus. Diese sind bei uns gespeichert und der HTML-Code wird so generiert, dass das Banner über unsere Seite geladen wird, wenn du den Code korrekt einbaust.
+Du kannst dir dein Partnerprogramm-Guthaben entweder in ZAP-Coins oder in echtes Geld auszahlen lassen. Gehe dazu wieder auf die [Affiliate-Seite](https://zap-hosting.com/en/customer/affiliate/) und klicke auf den blauen Button **Auszahlung/Transfer**.
 
-Du kannst das Banner natürlich auch selbst in deinen Kanal oder auf deine Webseite hochladen und es dort mit deinem Affiliate-Link verknüpfen. Dafür speichere das gewünschte Banner in der Vorschau einfach per Rechtsklick-Speichern bei dir ab.
-:::
+![image](https://screensaver01.zap-hosting.com/index.php/s/639MHYsCXnf7YeP/preview)
 
-`Ziel`
+Folge nun dem entsprechenden Abschnitt unten, um in eine der beiden Währungen zu überweisen.
 
-:::info
-Du kannst die Seite, auf die dein Banner-Link weiterleitet, selbst aussuchen. Wenn du beispielsweise einen Minecraft-Videokanal betreibst, kannst du den Link einstellen, dass er auf unsere "[Minecraft Server mieten](https://zap-hosting.com/de/minecraft-server-mieten/)"-Seite weiterleitet.
-
-Generell empfehlen wir, unsere [Startseite](https://zap-hosting.com/de/) als Ziel zu verwenden
-:::
+### ZAP-Münzen
```

Comment:
don't we use `ZAP-Coins` in German as well?

Reply:
Yes, the German stuff is DeepL'd first so it may have done some weird things. Florian will review the translations soon. 👍